solr/solr-ref-guide/modules/upgrade-notes/pages/major-changes-in-solr-10.adoc

```diff
@@ -114,11 +114,13 @@ Users who that don't need to vary JAR access on a per-core basis have several op
 * The `addHttpRequestToContext` option in `solrconfig.xml` has been removed; it's obsolete.
 Nowadays, the HTTP request is available via internal APIs: `SolrQueryRequest.getHttpSolrCall().getReq()`.
 
-* JMX, SLF4J and Graphite metric reporters have been removed. Users should migrate to using OTLP or the /admin/metrics endpoint with external tools to get metrics to their preferred backend.
-
 === Security
 
 * There is no longer a distinction between trusted and untrusted configSets; all configSets are now considered trusted. To ensure security, Solr should be properly protected using authentication and authorization mechanisms, allowing only authorized users with administrative privileges to publish them.
 
 === Upgrade to Jetty 12.x
-Solr upgraded to Jetty 12.x from 10.x as Jetty 10 and 11 have reached end-of-life support. Jetty 12.x requires Java 17 or newer and is fully compatible with Solr's new minimum requirement of Java 21. This upgrade brings support for modern HTTP protocols and adopts the Jakarta EE 10 namespace. For more details, see https://webtide.com/jetty-12-has-arrived/.
\ No newline at end of file
+Solr upgraded to Jetty 12.x from 10.x as Jetty 10 and 11 have reached end-of-life support. Jetty 12.x requires Java 17 or newer and is fully compatible with Solr's new minimum requirement of Java 21. This upgrade brings support for modern HTTP protocols and adopts the Jakarta EE 10 namespace. For more details, see https://webtide.com/jetty-12-has-arrived/.
+
+=== Open Telemetry
+
+* The Prometheus exporter JMX, SLF4J and Graphite metric reporters have been removed. Users should migrate to using OTLP or the /admin/metrics endpoint with external tools to get metrics to their preferred backend.
```

Comment:
insert a comma inbetween "exporter" and "JMX" as they are separate things.

> Add a link to the OTEL Collector.

Do you no longer feel even referencing it is good? I think a suggestion of some tool is better than vaguely saying "external tools".